cli/sub_command.go

```diff
@@ -74,6 +76,8 @@ func configureSubCommand(app commandHost) {
 	act.Flag("last-per-subject", "Deliver the most recent messages for each subject in the Stream (requires JetStream)").UnNegatableBoolVar(&c.deliverLastPerSubject)
 	act.Flag("stream", "Subscribe to a specific stream (required JetStream)").PlaceHolder("STREAM").StringVar(&c.stream)
 	act.Flag("ignore-subject", "Subjects for which corresponding messages will be ignored and therefore not shown in the output").Short('I').PlaceHolder("SUBJECT").StringsVar(&c.ignoreSubjects)
+	act.Flag("slow-consumer", "Mimic a slow consumer").Default("0s").DurationVar(&c.slowConsumer)
+	act.Flag("slow-consumer-async", "Continue processing messages").Default("false").BoolVar(&c.slowConsumerAsync)
```

Comment:
What is the use case for this? 

I think this tool is not for mimicing broken behavior, it will confuse users.